### ISSUE TITLE
Remove pcall_wrapper in 1.7

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#1225]
Please remove pcall_wrapper from src/lua/init.lua and restore orignal LuaJIT implementation.

A test case:

```
tarantool> status, e = pcall(box.error, 1, 'xx')                                                                                        
---
...

tarantool> tostring(ffi.typeof(e))                                                                                                      
---
- ctype<const struct error &>
...

tarantool> e:unpack()
---
- type: ClientError
  message: Illegal parameters, xx
  trace:
  - file: '[C]'
    line: 4294967295
...
```

**Comment by rtsisyk** [comment]
A breaking change, must be done ASAP.

**Comment by rtsisyk** [comment]
Fixed by 935e5681e73d8836b8e88a64c2cdc26811290458
